src/components/Navbar/index.tsx

```diff
@@ -32,17 +35,40 @@ export class NavbarBase extends React.Component<Props> {
     dispatch(_requestLogOut());
   };
 
+  renderCommentsNavBar() {
+    const { comments } = this.props;
+
+    if (!comments || comments.length === 0) {
+      return null;
+    }
+
+    return (
+      <>
+        <div className={styles.infoItem}>
+          {gettext('Comments:')}
+          <div className={styles.commentCount}>{comments.length}</div>
+        </div>
+        <div className={styles.infoItem}>
+          <CommentSummaryButton />
+        </div>
+      </>
+    );
+  }
+
   render() {
     const { user, currentVersion } = this.props;
 
     return (
-      <Navbar bg="dark" className={styles.Navbar} expand="lg" variant="dark">
+      <Navbar className={styles.Navbar} expand="lg" variant="dark">
         <Navbar.Brand className={styles.brand}>
-          {currentVersion && (
-            <span className={styles.addonName}>
-              {getLocalizedString(currentVersion.addon.name)}
-            </span>
-          )}
+          <div className={styles.info}>
+            {currentVersion && (
+              <div className={styles.infoItem}>
```

Comment:
I notice in the test that you used to check for the text of `.${styles.addonName}`, which would only include the add-on's name, but now you're checking for the text of `.${styles.info}` which could include more than the name. I know you're controlling that in the test by making sure there aren't also some comments, but do you think there is any value in still assigning a unique `className` to this `div`? I'm not sure there is, but figured I'd raise it.

Reply:
I try to avoid adding class names purely for testing so that's my one hesitation. If we ever show the comments navbar for zero or undefined comments then this test would break but we could cross this bridge when we come to it.